tasks/build/build.js

```diff
@@ -50,19 +52,23 @@ const pack = () => {
   childProcess.execSync(`npm run pack`, { stdio: `inherit` })
 }
 
-function sha256File(path) {
-  let hash = createHash("sha256")
-  fs.createReadStream(path).pipe(hash)
-  return new Promise((resolve, reject) => {
-    hash.on("data", hash => resolve(hash.toString("hex")))
-  })
+const sha256 = async data => {
```

Comment:
wouldn't `async function sha256(data) {...}` tell us better what this statement does?

Reply:
That question is blog-worthy!

https://www.nodeguy.com/javascript-function-definitions/

Reply:
Unification of the syntax is not always better for readability: https://en.wikipedia.org/wiki/Brainfuck
Hoisting sounds like a good reason to use this syntax. Which is to be valued higher, I don't know.
Let's keep it this way.

Reply:
Your Brainfuck point is an excellent rebuttal.

I still prefer the expression syntax because it makes it easier to modify functions later with higher-order modifiers.

Reply:
You have an example?

Reply:
Not off the top of my head.  I'll let you know the next time I run across one.